ggml/src/ggml-sycl/dpct/helper.hpp

```diff
@@ -2426,6 +2426,7 @@ namespace dpct
                                            b, ldb, beta, c, ldc, batch_size);
             break;
         }
+#endif
```

Comment:
Do you think it is worth creating an issue to inform the DPCT repo regarding the change in available dtypes?

Reply:
Good idea @AidanBeltonS